bogota-abril-2019/cyclomatic-complexity/david-villamizar/predict-age.js

```diff
@@ -0,0 +1,12 @@
+function predictAge(age1, age2, age3, age4, age5, age6, age7, age8) {
+  const sum =
+    age1 * age1 +
+    age2 * age2 +
+    age3 * age3 +
+    age4 * age4 +
+    age5 * age5 +
+    age6 * age6 +
+    age7 * age7 +
+    age8 * age8;
```

Comment:
Might be cleaner and more scalable!
`Array.prototype.slice.call(arguments);`

Check 
[arguments](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments)

Reply:
Indeed, I know it can be done with arguments. I did it this way to earn a lower cyclomatic complexity score.

Reply:
A solution using `arguments` could be:
```suggestion
     const sum = Array.prototype.slice.call(arguments).map(age => age * age).reduce((sum, ageSq) => sum + ageSq);
```